### ISSUE TITLE
add actions props to the ApiExplorerPage like in the CatalogIndexPage

### ISSUE THREAD

**Issue by Madebymaurice** [backstage/backstage#7452]
In the `CatalogIndexPage`, you've had the `columns` and `actions` props, which help to customize the columns and the actions in the `CatalogTable`. In the `ApiExplorerPage` there is also the `columns` props available but not the actions one.

## Feature Suggestion

Just like in the `CatalogIndexPage` add the `actions` props in the `ApiExplorerPage`

## Possible Implementation

```
export type ApiExplorerPageProps = {
  initiallySelectedFilter?: UserListFilterKind;
  columns?: TableColumn<CatalogTableRow>[];
  actions?: TableProps<CatalogTableRow>['actions'];
};

export const ApiExplorerPage = ({
  initiallySelectedFilter = 'all',
  actions,
  columns,
}: ApiExplorerPageProps) => {
```

## Context

I'll submit a PR

**Comment by freben** [comment]
Hard to say without the code. Could be many reasons - for example if there's an actual error in your code, then maybe an error is being rendered instead of the expected table. Or maybe something that's asynchronous isn't be ing awaited so the final state hasn't landed yet.

**Comment by Madebymaurice** [comment]
Yes I understand, I've created the PR with the failed tests, it will be easier to test for you, seems that I'm not able to link the issue to the PR, so here is the link https://github.com/backstage/backstage/pull/7518 (sorry, i've done Github stuff since long time ^^)

**Comment by benjdlambert** [comment]
Merged in #7518
